file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
+                puppet.bat *                  # executes Puppet and provides results of Puppet runs
```

Comment:
Thanks @ferventcoder .
Will avoid adding this to the specs and leave just the directory, as suggested by @kylog.